presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -239,13 +239,22 @@ public LateralJoinNode lateral(List<Symbol> correlation, PlanNode input, PlanNod
 
     public TableScanNode tableScan(List<Symbol> symbols, Map<Symbol, ColumnHandle> assignments)
     {
+        return tableScan(symbols, assignments, null);
+    }
+
+    public TableScanNode tableScan(List<Symbol> symbols, Map<Symbol, ColumnHandle> assignments, Expression originalConstraint)
+    {
         TableHandle tableHandle = new TableHandle(new ConnectorId("testConnector"), new TestingTableHandle());
-        return tableScan(tableHandle, symbols, assignments);
+        return tableScan(tableHandle, symbols, assignments, originalConstraint);
     }
 
     public TableScanNode tableScan(TableHandle tableHandle, List<Symbol> symbols, Map<Symbol, ColumnHandle> assignments)
     {
-        Expression originalConstraint = null;
+        return tableScan(tableHandle, symbols, assignments, null);
+    }
+
+    public TableScanNode tableScan(TableHandle tableHandle, List<Symbol> symbols, Map<Symbol, ColumnHandle> assignments, Expression originalConstraint)
```

Comment:
`originalConstraint` doesn't sound like `filter` (mentioned in the cmt msg). Oh, and the `originalConstraint` field bears the following comment, are you sure you want to set it here?

> HACK! This field exists for the sole purpose of being able to print the original predicates (from the query) in a human readable way.